website/tailwind.config.ts

```diff
@@ -32,6 +32,7 @@ const config: Config = {
       },
     },
   },
+  content: [...theGuildTailwindConfig.content, './theme.config.tsx'],
```

Comment:
`max-w-[1392px]` didn't work so the navigation on landing didn't align with the hero and looked kinda off on large screens